README.md

```diff
@@ -30,13 +31,18 @@ one you should use, refer to the [`argon2` repo][argon2].
 var argon2i = require('argon2-ffi').argon2i;
 // var argon2d = require('argon2-ffi').argon2d; if you'd like to use argon2d
 
-var password = new Buffer('password1');
-var salt = new Buffer('saltysalt');
-argon2i.hash(password, salt, function(err, res) {
-  console.log(res); // $argon2i$v=19$m=4096,t=3,p=1$c2FsdHlzYWx0$oG0js25z7kM30xSg9+nAKtU0hrPa0UnvRnqQRZXHCV8
+var password = 'password1'; // Can also be a Buffer
+crypto.randomBytes(128, function (err, salt) {
```

Comment:
`128` bytes is needlessly excessive for a salt. OWASP, for example, recommends 8 bytes for . CrackStation recommends keeping the salt length the same as the hash length (in the default case of `argon2-ffi`, that's just 32 bytes).

https://www.owasp.org/index.php/Password_Storage_Cheat_Sheet
https://crackstation.net/hashing-security.htm

I think 32 bytes is probably the best recommendation for the README.